### DISCUSSION TITLE
Theme assets mixing up

### DISCUSSION THREAD

**Discussion by MikeKry** [OrchardCMS/OrchardCore#12094] [Q&A]
I have encountered a problem during development. 

I am running several OC projects, each on different version of OrchardCore, including two projects that have nuget packages build from source manually and stored in filesystem. Everything worked ok, until I dowloaded newer version of OrchardCore (1.4.0) into one of the projects that includes manually build nugets.

After that, css and js broke on older version of OC (1.4-preview). After long investigation, I have figured out, that both solutions include the same css and js files, even though I have updated only one of the projects.

Both of them have bootstrap 5.1.3 in TheAdmin.css but the older one needs bootstrap 4.6.1 etc.. 

So the current state is:
Project 1 - updated version of nugets to 1.4.0 - working fine
Project 2 - older version 1.4.0-preview - broken, somehow includes css and js from Project 1
other projects - 1.3.x versions - working ok.

Also, we have tested it on 3 computers:
1) works ok, both projects
2) broken, both projects include newer version of assets
3) broken, both projects include older version of assets

Do you have any idea what could cause this?
I have tried:
* cleaning solution
* cleaning nuget cache, iis 
* cloning project repo again
* rebuilding all nugets again
* decompiling both nuget versions (and they include correct versions of assets)

Version of TheAdmin.css from browser:
![image](https://user-images.githubusercontent.com/52829889/180854365-0e76457a-955d-4002-9a53-a2fde1279e85.png)

decompiled version from nuget:
![image](https://user-images.githubusercontent.com/52829889/180856339-7a4197c8-df7d-4335-b7a7-19b12ac0920a.png)

**Comment by MikeKry** [comment]
Solved... 
Built nuget packages contain path to folder with OrchardCore repo, and it seems that dynamic module loading fetches assemblies from this repo instead of Bin/Debug folder, if it is available. (the two PCs have OC stored in D:/DEV, third PC does not. When path does not exist, it works ok, otherwise it takes files from folder inside this repo).

Maybe worth checking if it could be a problem?

**Reply by ns8482e** [reply]
It’s default behavior, if module is referenced as project source then assets are served from file system if environment is Development 

If environment is not a Development then assets are served from assembly
